instance/models/openedx_instance.py

```diff
@@ -127,7 +127,7 @@ def database_name(self):
         # Escape all non-ascii characters and truncate to 64 chars, the maximum for mysql:
```

Comment:
This comment is now outdated, we truncate to 50 chars. I think it would be good to explain why we are truncating to 50 rather than 64 chars.